Console/Command/RemoveUnusedMediaCommand.php

```diff
@@ -71,6 +71,16 @@ public function execute(InputInterface $input, OutputInterface $output) : void
 
         $directoryIterator = new \RecursiveDirectoryIterator($imageDir);
 
+        $imagesToKeepArray = $connection->fetchAll(
```

Comment:
I think you could save the `foreach` by using `fetchCol` here.

Reply:
Code now updated. Thanks for the hint. Hadn't actually used fetchCol before